@xen-orchestra/web-core/lib/components/ui/input/UiInput.vue

```diff
@@ -1,41 +1,39 @@
 <!-- v5 -->
 <template>
-  <div class="ui-input" :class="toVariants({ accent })">
-    <UiLabel v-if="slots.default || label" :accent="labelAccent" :required :icon="labelIcon" :href :for="id">
-      <slot>{{ label }}</slot>
-    </UiLabel>
-    <div>
-      <VtsIcon :icon accent="current" class="before" />
-      <input
-        :id
-        v-model.trim="modelValue"
-        class="typo-body-regular input text-ellipsis"
-        :type
-        :disabled
-        v-bind="attrs"
-      />
-      <VtsIcon
-        v-if="!attrs.disabled && modelValue && clearable"
-        :icon="faXmark"
-        class="after"
-        accent="brand"
-        @click="modelValue = ''"
-      />
-    </div>
-    <UiInfo v-if="slots.info || info" :accent="accent === 'brand' ? 'info' : accent">
-      <slot name="info">{{ info }}</slot>
-    </UiInfo>
+  <div :class="toVariants({ accent, disabled })" class="ui-input" @click.self="focus()">
+    <VtsIcon :icon accent="current" class="left-icon" />
+    <input
+      :id="wrapperController?.id ?? id"
+      ref="inputRef"
+      v-model.trim="modelValue"
+      :disabled
+      :required
+      :type
+      class="typo-body-regular input text-ellipsis"
+      v-bind="attrs"
+    />
+    <VtsIcon
+      v-if="!disabled && modelValue && clearable"
+      :icon="faXmark"
+      accent="brand"
+      class="clear-icon"
+      @click="clear()"
+    />
+    <slot v-if="slots.rightIcon || rightIcon" name="rightIcon">
```

Comment:
Based on our discussion in Mattermost, we should use _kebab-case_ for slots names. I will add it to our guidelines.
```suggestion
    <slot v-if="slots['right-icon'] || rightIcon" name="right-icon">
```